package.json

```diff
@@ -67,8 +67,8 @@
         "react-dom": "^16.8"
     },
     "dependencies": {
-        "@dhis2/d2-i18n": "1.0.5",
-        "@dhis2/ui-core": "4.0.0-beta.2",
+        "@dhis2/d2-i18n": "^1.0.5",
+        "@dhis2/ui-core": "^4.0.1",
```

Comment:
I'm thinking, should we not make this a peerdependency? Seems more straightforward to me to treat this lib as an addon to ui-core, and to leave installing ui-core to the person using ui-forms.

Reply:
Although this is could be a good idea, for now I think we shouldn't change this in the current PR. The current setup matches that in `ui-widgets` and I think we should keep these two packages identical. It might be good to create an issue to discuss peer vs direct dependencies and, once the discussion is resolved, address this in both packages.

Reply:
Yeah, plus it might not match with how we approach our libs, so discussing it separately sounds like a good idea 👍

Reply:
changing so that `ui-core` is a peer dep of `ui-widgets` has been on my radar for a long time now so I second that decision here.

Reply:
I've created these two issues, so we don't forget:
- https://github.com/dhis2/ui-forms/issues/141
- https://github.com/dhis2/ui-widgets/issues/229

@varl just to confirm... We should move both `@dhis2/ui-core` and `@dhis2/d2-i18n` to peer dependencies, right?